packages/core/src/auth/sso/ssoAccessTokenProvider.ts

```diff
@@ -454,15 +473,20 @@ export class DeviceFlowAuthorization extends SsoAccessTokenProvider {
      * created and returned.
      */
     override async getValidatedClientRegistration(): Promise<ClientRegistration> {
-        const cacheKey = this.registrationCacheKey
-        const cachedRegistration = await this.cache.registration.load(cacheKey)
-
-        // Clear cached if registration is expired
-        if (cachedRegistration && isExpired(cachedRegistration)) {
-            await this.invalidate('registrationExpired:DeviceCode')
-        }
+        return telemetry.function_call.run(
```

Comment:
Is it possible for these next 3 functions to use the decorator? We could optionally add emit as one of the input options

Reply:
For some odd reason the decorator was causing a `Reference error` in the `create()` method due to the subclasses not being found. This happened in CI but not locally.

This is a quick fix to that